src/store/reducers/MovieReducer.js

```diff
@@ -1,12 +1,14 @@
 import { SET_MOVIES } from '../actions/ActionTypes';
 
 const initialState = {
-  all: []
+  movies: {
+    page: 1
+  }
```

Comment:
ovdje sobizrom da znas kakav response dolazi kada se radi u pitanju paginacija, mozes staviti neki inital state, cisto da se zna koje su to sve informacije koje ces imati ovdje